src/OrchardCore.Modules/OrchardCore.Admin/Admin/AdminSettings.cs

```diff
@@ -2,8 +2,8 @@ namespace OrchardCore.Admin.Models
 {
     public class AdminSettings
     {
-        public bool DisplayMenuFilter { get; set; }
+        public bool DisplayMenuFilter { get; set; } = true;
 
-        public bool DisplayDarkMode { get; set; }
+        public bool DisplayDarkMode { get; set; } = true;
```

Comment:
@agriffard as the property said display the dark mode but not enable it, so we 're safe ;)